WordPress/openverse-frontend
# Enable audio interactions on the audio track component

## Fixes


Fixes #1297 by @sarayourfriend 

## Description



This PR builds upon work done by @sarayourfriend. It enables audio track controls using the keyboard when the audio track is focused and makes the subcomponents non-focusable to provide accessible navigation in the search results.

## Testing Instructions

0. Check out the PR and run the dev server.
1. Search for any given term and switch to the audio-only results.
2. Navigate using the <kbd>Tab</kbd> key. The focus should move from one result track to the next and not go inside the component.
3. Manipulate playback by seeking (with arrows <kbd>&larr;</kbd> and <kbd>&rarr;</kbd>) and toggling (with the <kbd>Space</kbd>) key.
4. Try to access the same functionality with the mouse by clicking the buttons and dragging the seekbar etc. Existing functionality should continue to work. 

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
